app/assets/stylesheets/common/admin/users.scss

```diff
@@ -149,6 +149,18 @@
         }
       }
     }
+
+    &__cell.silence_reason {
+      text-align: left;
```

Comment:
Can we make the header left-aligned, too?

<img width="277" alt="Screenshot 2025-01-08 at 12 23 47 PM" src="https://github.com/user-attachments/assets/efd84392-0e60-452b-a926-cbd5fe3108a5" />

Reply:
I think I will leave this to @ellaestigoy cause  there is a far bigger more complicated change needed here. 

The whole table falls apart at 10 columns and there is no underline on the bulk selector, something about the pattern we are doing tables is off, I feel like we need a different control here.